src/cmds/build/bundle.ts

```diff
@@ -1,37 +1,36 @@
-const { promises: fs, createWriteStream } = require('fs');
-const browserify = require('browserify');
-const stripComments = require('strip-comments');
-// const terser = require('terser')
+import { promises as filesystem, createWriteStream } from 'fs';
+import { logError } from '../../utils';
+import browserify from 'browserify';
+import stripComments from 'strip-comments';
 
-const { logError } = require('../../utils');
-
-module.exports = { bundle };
+/* Custom Type Imports */
+import { Argument, Option } from '../../types/yargs';
 
 /**
  * Builds a Snap bundle JSON file from its JavaScript source.
  *
  * @param {string} src - The source file path
  * @param {string} dest - The destination file path
- * @param {object} argv - argv from Yargs
+ * @param {Argument} argv - arguments as an object generated by yargs
  * @param {boolean} argv.sourceMaps - Whether to output sourcemaps
  * @param {boolean} argv.stripComments - Whether to remove comments from code
  */
```

Comment:
All [JSDoc](https://jsdoc.app/) type annotations, e.g. `{string}` or `{object}`, in docstring comments should be removed in all TypeScript files.

JSDoc type annotations are useful in plain JavaScript files, but useless and potentially misleading / harmful in TypeScript files.